tests/v2/validation/resource/terraform/v1charts/target/main.tf

```diff
@@ -0,0 +1,149 @@
+#Target cluster for v1 charts installation
+terraform {
+  required_version = ">= 0.14"
+  required_providers {
+    rancher2 = {
+      source = "rancher/rancher2"
+    }
+  }
+}
+
+provider "rancher2" {
+  api_url   = var.rancher_api_url
+  token_key = var.rancher_token_key
+}
+
+resource "rancher2_cluster_sync" "v1target-sync" {
+  cluster_id =  var.cluster_id
+}
+
+resource "local_file" "sync_kube_config" {
+    depends_on = [rancher2_cluster_sync.v1target-sync]
+    content  = rancher2_cluster_sync.v1target-sync.kube_config
+    filename = "kube_config_provisioned_cluster.yaml"
+}
+
+
+resource "rancher2_namespace" "longhorn-system-namespace" {
+  depends_on = [rancher2_cluster_sync.v1target-sync,local_file.sync_kube_config]
+  name = "longhorn-system"
+  project_id = var.project_id
+  description = "longhorn-system namespace"
+}
+
+
+resource "null_resource" "longhorn-iscsi-nfs" {
+  depends_on = [rancher2_cluster_sync.v1target-sync,local_file.sync_kube_config,rancher2_namespace.longhorn-system-namespace]
+  provisioner "local-exec" {
+    environment = {
+      KUBECONFIG = "kube_config_provisioned_cluster.yaml"
+    }
+    command = "kubectl apply -f https://raw.githubusercontent.com/longhorn/longhorn/v1.3.0/deploy/prerequisite/longhorn-iscsi-installation.yaml --namespace=longhorn-system && kubectl apply -f https://raw.githubusercontent.com/longhorn/longhorn/v1.3.0/deploy/prerequisite/longhorn-nfs-installation.yaml --namespace=longhorn-system"
```

Comment:
Should this be hardcoded? Will we ever want to change the version?

Reply:
Added Longhorn prereq variable and set current defaults for 2.5 and 2.6
	- This removes the hardcoded version number